build/memory_tests.ps1

```diff
@@ -0,0 +1,14 @@
+<# dotMemoryUnit requires the pull path of nunit3-console.
```

Comment:
The script below wasn't making much sense to me until I read the comment specifying that `dotMemoryUnit` requires the full path. 👍